src/domain/release-archive.ts

```diff
@@ -53,13 +53,22 @@ export class ReleaseArchive {
     const stripComponents = this.stripPrefix
       ? this.stripPrefix.split("/").length
       : 0;
+
+    let modulePath = path.join(this.stripPrefix, "MODULE.bazel");
```

Comment:
nit: seems like this assignment should be in an `else` case so modulePath could be const?